### PR TITLE
Reacting to verbose rename

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Reacting to https://github.com/aspnet/Logging/pull/314.
Tested locally.

### CODE REVIEW

File: src/Microsoft.AspNet.DataProtection/LoggingExtensions.cs [hunk 1]
```diff
@@ -421,13 +421,13 @@ public static bool IsInformationLevelEnabled(this ILogger logger)
...
         /// <summary>
-        /// Returns a value stating whether the 'verbose' log level is enabled.
+        /// Returns a value stating whether the 'trace' log level is enabled.
         /// Returns false if the logger instance is null.
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool IsVerboseLevelEnabled(this ILogger logger)
+        public static bool IsTraceLevelEnabled(this ILogger logger)
         {
-            return IsLogLevelEnabledCore(logger, LogLevel.Verbose);
```
Comment (kichalla) [inline src/Microsoft.AspNet.DataProtection/LoggingExtensions.cs]: Why is this not `Debug`? or an intentional behavior change?

Comment (kichalla) [inline src/Microsoft.AspNet.DataProtection/LoggingExtensions.cs]: Based on your other PR (logging) comment, this makes sense to me now...but the above `LoggingExtensions.cs` changes rename `Verbose` to `Debug`?

Comment (JunTaoLuo) [inline src/Microsoft.AspNet.DataProtection/LoggingExtensions.cs]: This is an api change. There is already an `IsDebugLevelEnabled` function. The only thing that needs to change here is `Verbose` to `Trace` so every level has a corresponding helper function.

**Comment by JunTaoLuo** [comment]
@kichalla I was wondering if you still had questions for this one?

**Comment by kichalla** [comment]
:shipit:

**Pull request merged**